charts/i2b2/values.yaml

```diff
@@ -0,0 +1,89 @@
+# Default values for i2b2.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: i2b2/i2b2-web
```

Comment:
their base image is:

https://hub.docker.com/layers/library/centos/centos7.9.2009/images/sha256-dead07b4d8ed7e29e98de0f4504d87e8880d4347859d839686a31da35a3b532f?context=explore

![Screenshot 2024-09-13 at 16-06-28 Image Layer Details - centos centos7 9 2009](https://github.com/user-attachments/assets/c4e56aea-363a-4747-8364-79e8e104f27d)

😨

Reply:
The only image provided for i2b2

Reply:
I guess it's on their todo list :) I agree that there is nothing you can do here aside telling them that they are on an EOL'd base image.

Reply:
😨😨😨